app/src/lib/email.ts

```diff
@@ -9,36 +9,35 @@ import { Account } from '../models/account'
  *
  *  - the primary email if it's publicly visible
  *  - the first public email
```

Comment:
Not related to this PR, but if I understand the logic correctly, we don't seem to be looking for the first public email, so maybe we can take the opportunity to change the logic or update the comment.

Reply:
Sounds good 👇 

```suggestion
```